PokemonUnity.Shared/Monster/Pokemon/Data/Base.cs

```diff
@@ -0,0 +1,298 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using PokemonUnity;
+using PokemonUnity.Monster;
+using PokemonUnity.Attack;
+using PokemonUnity.Inventory;
+
+namespace PokemonUnity.Monster.Data
+{
+	public struct PokemonData
```

Comment:
I really enjoy this change, if i'm being honest... I think it really helps to decouple the tangles of the code, and helps things to breathe a little better.